src/display.c

```diff
@@ -186,6 +213,30 @@ size_t LCUIDisplay_Render(void)
 		}
 		RectList_Clear(&rects);
 		RectList_Clear(&record->rects);
+
+		UpdateFlashRects(record);
+		for (LinkedList_Each(fr, &record->flashRects)) {
```

Comment:
This `for` loop code can be defined as a function, we should keeps `LCUIDisplay_Update()` simple.